openbb_terminal/stocks/stocks_helper.py

```diff
@@ -1044,3 +1060,33 @@ def verify_plot_options(command: str, source: str, plot: list) -> bool:
                 )
         return True
     return False
+
+
+def heikin_ashi(data: pd.DataFrame) -> pd.DataFrame:
+    """Return OHLC data as Heikin Ashi Candles.
+
+    Parameters
+    ----------
+    data: pd.DataFrame
+        DataFrame containing OHLC data.
+
+    Returns
+    -------
+    pd.DataFrame
+        Appended DataFrame with Heikin Ashi candle calculations.
+    """
+
```

Comment:
can you add a check that the correct column names are in the data, that way accessing data["Open"] doesnt throw an error if data["open"] sneaks in

Reply:
Added a check for this, and attempts to rename columns if they are lower case. If the correct columns are still not in the DataFrame a ValueError is raised.